backend/development.ini.sample

```diff
@@ -388,12 +411,14 @@ website.base_url = %(basic_setup.website_base_url)s
 ### IMAP ###
 # Tracim email reply feature will read all mails in following IMAP,
 # to be able to transform email responses into Tracim app comments.
-; email.reply.imap.server = your_imap_server
-; email.reply.imap.port = 993
-; email.reply.imap.user = your_imap_user
-; email.reply.imap.password = your_imap_password
+
+# email.reply.imap.server = your_imap_server
+# email.reply.imap.port = 993
+# email.reply.imap.user = your_imap_user
+# email.reply.imap.password = your_imap_password
```

Comment:
Same for parameter here:
- `; email.reply.imap.server = `
- `; email.reply.imap.port = `
- `; email.reply.imap.user = `
- `; email.reply.imap.password = `

Reply:
fixed